README.md

```diff
@@ -109,6 +116,44 @@ from djangoevents import subscribe_to                   # from eventsourcing.dom
 from djangoevents import EventSourcedRepository         # from eventsourcing.infrastructure.event_sourced_repo import EventSourcedRepository
 ```
 
+### Documenting event schema
```

Comment:
maybe add some documentation of the meaning and how to use the abstract decorator?